console/src/main/java/org/jline/console/impl/DefaultPrinter.java

```diff
@@ -559,9 +560,8 @@ private String columnValue(String value) {
     @SuppressWarnings("unchecked")
     private Map<String,Object> objectToMap(Map<String, Object> options, Object obj) {
         if (obj != null) {
-            Map<Class<?>, Object> toMap = options.containsKey(Printer.OBJECT_TO_MAP)
-                                                 ? (Map<Class<?>, Object>)options.get(Printer.OBJECT_TO_MAP)
-                                                 : new HashMap<>();
+            Map<Class<?>, Object> toMap = (Map<Class<?>, Object>)
+                    options.getOrDefault(Printer.OBJECT_TO_MAP, Collections.emptyMap());
             if (toMap.containsKey(obj.getClass())) {
```

Comment:
This changes the return value from `new HashMap<>()` to `Collections.emptyMap()`, the last one being unmodifiable.  Will that cause any issue to callers ?

Reply:
Not sure I get this one. I did update only of `toMap`. And `toMap` is used only in this method.

Reply:
My bad, I misread.